electricpy/__init__.py

```diff
@@ -4782,6 +4782,102 @@ def wireresistance(length=None,diameter=None,rho=16.8*10**-9,R=None):
     elif R != None and length != None:
         A = rho*length/R
         return _np.sqrt(4*A/pi)
+
+def parallel_plate_capacitance(A=None, d=None, e=e0, C=None):
+    r"""
+    Parallel-Plate Capacitance Calculator.
+
+    Enter three values to calculate the remaing one. Even though every variable
+    is unitless, please use the International System of Units.
+
+    .. math:: C =\frac{\varepsilon A}{d}
+
+    Parameters
+    ----------
+    A:  [float]
```

Comment:
The square brackets are not explicitly needed to denote parameters of a single type (float, in this case). Although it's fine to use them, this practice would seem inconsistent with the rest of the library.

- [ ] remove square brackets from parameter type-strings

Reply:
It's a pleasure to collaborate! I like the changes you propose. I was relying on the `wireresistance` function to write the documentation for `parallel_plate_capacitance` and `solenoid_inductance`. However, I noticed that in the `wireresistance` function, there are parameters indicated as: `[float], optional`. That is, they are also indicated with brackets. Additionally, I just realized that I forgot to put that the parameters are optional. I then propose to place `float, optional`.

Reply:
That looks fantastic! You know, you're probably right! I bet the `wireresistance` function should be updated to change that too. We typically use the `[]` brackets when there are two or more types that could be used; something like: `[str, float]`

Thanks for tackling this! :)